lib/gpu_memory_service/tests/test_persistent_kv_defaults.py

```
@@ -0,0 +1,69 @@
+        lambda device, tag: f"socket:{device}:{tag}",
+    )
+
+    assert utils.get_gms_persistent_kv_socket(7, "GMS_VLLM_VMM_IPC_SOCKET") == (
+        "socket:7:kv_cache"
+    )
+
+
+def test_persistent_kv_socket_allows_explicit_override(monkeypatch):
+    from gpu_memory_service.integrations.common import utils
+
+    monkeypatch.setenv("GMS_VLLM_VMM_IPC_SOCKET", "/tmp/explicit.sock")
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/explicit.sock"

(S108)